source/includes/_customers.md

```diff
@@ -0,0 +1,286 @@
+# Customers
+
+## user/status
+
+> &gt;&gt;&gt; Script
+
+```python
+[...]
+import requests
+
+work_provider_user_id = "12345"
+
+response = requests.get(
+    f'https://api.zego.com/v1/user/status/?workProviderUserId={work_provider_user_id}',
+    headers={'Authorization': AUTHORIZATION_CODE},
```

Comment:
Is it worth adding a section on authentication? We don't explicitly say how to authenticate anywhere, we just show it in the examples.

Reply:
I put a tiny little section on the introduction page on Authentication / Authorisation